build-test-elasticsearch7.xml

```diff
@@ -1129,6 +1376,32 @@ xpack.monitoring.collection.enabled: true</echo>
 
 				<lstopwatch action="total" name="start.elasticsearch" />
 			</then>
+			<elseif>
+				<equals arg1="${elastic.monitoring.pki.authentication.enabled}" arg2="true" />
+				<then>
+					<sleep seconds="30" />
+
+					<set-x-pack-user-passwords />
+
+					<start-x-pack-trial />
+
+					<sleep seconds="10" />
+
+					<antcall target="stop-elasticsearch" />
+
+					<enable-pki-properties />
```

Comment:
This needs a name that more clear, but after refactoring it'll be more clear how we should name our pki macrodefs